### DISCUSSION TITLE
This project needs PROJECT document.

### DISCUSSION THREAD

**Discussion by qqkookie** [diasurgical/DevilutionX#1961] [General]
I think that this project needs some PROJECT overview  document to list future plan and record its progress.  There are docs/TODO.md and CHANGELOG.md but they are just programmer's coding memo, and change log, not planned feature or progress report, discussed by members. The project document will be maintained by project owner and major contributors.  Intended reader is potential contributor on this project or interested users.

The project  document will include project overview and leading members and general future direction where this project is heading, discussed and accepted by leading members of this project. And planned  features they are working on,  and ts progress and  release plan. or new feature they are considering to commence. It also includes how to suggest new feature by other people (user/game player) and list of suggested or desirable features / work item proposed by them, that need to be discussed.

Such project overview will prompt for more active user feedback and interest and coordinated participation of contributor . 

And as a starting point, I suggest to discuss on introducing some good QOL features of "Bezebub/Tchernobog HD MOD"(https://mod.diablo.noktis.pl/) into this project. I think that the developer of the MOD is participating here, too.

**Comment by AJenbo** [comment]
I see the pros of a document like this, but I'm not particularly good at writing and do not enjoy it so such a document would not be written by me (which seams to be the request). Such a document also run the risk of getting out of sync with the project and outdated.

If someone else would be interested in writing a document like this I'm happy to review it and make comments on it. I'm also happy to answer any questions regarding the project goals and status.

I'm not exactly sure what you are expecting the document to contain as fare as roadmap goes, but it's possible that you expect there to be a more clear plan then is actually the case, it isn't really set in stone more then what is stated in the readme: "strives to make it simple to run the game while providing engine improvements, bugfixes, and some optional quality of life features", what that exactly entails is a bit fluid and may change as things progress.

What's currently being worked on can be found under the current milestone https://github.com/diasurgical/devilutionX/milestones

Suggestions and discussions about ideas, features and requests can be made right here on GitHub Discussions, or via the Discord chat. This doesn't really seam related to a project document though.
